### ISSUE TITLE
Typo on 2.4.7 Focus Visible is marked as Level A (should be Level AA) on the quick ref.

### ISSUE THREAD

**Issue by rkanswers** [w3c/wai-wcag-quickref#249]
Typo Here:
https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#focus-visible

https://www.w3.org/TR/WCAG21/#focus-visible

**Comment by iadawn** [comment]
Thanks for the spot... addressed with #248
